Makefile

```diff
@@ -8,6 +8,10 @@ ifndef VERSION
 	VERSION := $(shell git describe --tags --always --dirty="-dev")
 endif
 
+ifndef TARGETARCH
+	TARGETARCH := $(shell arch)
```

Comment:
An issue we might have here is that OS X reports `arm64` but linux reports `aarch64` when running this command on the same machine and arch.

since *nix varieties cannot agree here i think we need to make sure this is going to output exactly what we expect it to.